underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -121,15 +124,17 @@ public void close() throws IOException {
   @Override
   public FSDataOutputStream create(String path) throws IOException {
     IOException te = null;
-    int cnt = 0;
-    while (cnt < MAX_TRY) {
+    RetryPolicy retryPolicy = new CountingRetry(MAX_TRY);
+    while (true) {
```

Comment:
`while(retryPolicy.attempRetry())` and then you can remove the logic at the end of the catch.